src/components/data_submission/DataAccessGovernance.js

```diff
@@ -0,0 +1,168 @@
+import ConsentGroupForm from './consent_group/ConsentGroupForm';
+import { useEffect, useState } from 'react';
+import { cloneDeep, isNil, every } from 'lodash/fp';
+import { div, h, h2, a, span } from 'react-hyperscript-helpers';
+import { DAC } from '../../libs/ajax';
+import { FormFieldTypes, FormField } from '../forms/forms';
+
+const OPEN_ACCESS = 'Open Access';
+const CLOSED_ACCESS = 'Closed Access';
+
+const openClosedRadioOptions =     [
+  {
+    id: 'open_access',
+    name: OPEN_ACCESS,
+    text: 'Open Access'
+  },
+  {
+    id: 'closed_access',
+    name: CLOSED_ACCESS,
+    text: 'Closed Access',
+  }
+];
+
+export const DataAccessGovernance = (props) => {
+  const {
+    onChange
+  } = props;
+
+  const [consentGroupsState, setConsentGroupsState] = useState([]);
+  const [dacs, setDacs] = useState([]);
+  const [isClosedAccess, setIsClosedAccess] = useState(false);
+
+  useEffect(() => {
+    DAC.list(false).then((dacList) => {
+      setDacs(dacList);
+    });
+  }, []);
+
+  useEffect(() => {
+    const filteredConsentGroupsState = consentGroupsState.filter(state => !isNil(state));
+
+    const groups = filteredConsentGroupsState.map(state => state.consentGroup);
+    const valid = every(filteredConsentGroupsState.map(state => (!state.editMode) && state.valid));
+
+    onChange({key: 'consentGroups', value: groups, isValid: valid});
+  }, [consentGroupsState, onChange]);
+
+  const addNewConsentGroup = () => {
+    const newConsentGroupsState = cloneDeep(consentGroupsState);
+    newConsentGroupsState.push({
+      consentGroup: {},
+      editMode: true,
+      valid: false,
+    });
+    setConsentGroupsState(newConsentGroupsState);
+  };
+
+  const deleteConsentGroup = (idx) => {
+    const newConsentGroupsState = cloneDeep(consentGroupsState);
+    newConsentGroupsState[idx] = undefined;
+    setConsentGroupsState(newConsentGroupsState);
+  };
+
+  const updateConsentGroup = (idx, value, isValid) => {
+    const newConsentGroupsState = cloneDeep(consentGroupsState);
+    newConsentGroupsState[idx] = {
+      editMode: false,
+      valid: isValid,
+      consentGroup: {
+        ...consentGroupsState[idx],
+        ...value,
+      }
+    };
+    setConsentGroupsState(newConsentGroupsState);
+  };
+
+  const startEditConsentGroup = (idx) => {
+    const newConsentGroupsState = cloneDeep(consentGroupsState);
+    newConsentGroupsState[idx].editMode = true;
+    setConsentGroupsState(newConsentGroupsState);
+  };
+
+  return div({
+    style: {
+      padding: '50px 0',
```

Comment:
I don't have access to the invision mock so I'm not sure if it's intentional but the space between the DataSubmissionStudyInformation and DataAccessGovernance sub-components looks kinda large to me. I think you'd have to change this padding and the padding in [ds_study_information](https://github.com/DataBiosphere/duos-ui/blob/6e5a46d32d51cf9e974aa9657d621d4212779318/src/components/data_submission/ds_study_information.js#L34) to make that gap smaller. This is purely a nit based on my own opinion so please let me know if I am way off!

Reply:
updated styling to be a bit more consistent / smaller spacing!